CIP-????/README.md

```diff
@@ -0,0 +1,104 @@
+---
+CIP: 127?
+Title: Dynamic Asset Resale Fee Standard
```

Comment:
```suggestion
Title: Dynamic Asset Resale Fees
```
... likewise we don't know if this will be a "Standard" so let's stick with something indisputable both now & later 🤓 (changed also in PR title so let's see how it works in both places)

Reply:
yes fair enough!